### ISSUE TITLE
Check the hyperlink colour contrast

### ISSUE THREAD

**Issue by giro090** [best-practice-and-impact/gptables#280]
Feedback from the Presentation Champions group is that the hyperlink blue colour may not meet accessibility contrast requirements, however users expect hyperlinks to be blue. The colour should be checked with a contrast checker tool, and a color of blue implemented with the necessary contrast.

Section 2 of the Analysis Function Spreadsheets Accessibility [Guidance](https://analysisfunction.civilservice.gov.uk/policy-store/releasing-statistics-in-spreadsheets/#section-2) stipulates that we should meet the AA level of the Web Content Accessibility Guidelines.

Definition of Done:
- Hyperlink blue colour on a white background is checked using a [contrast checker](https://webaim.org/resources/contrastchecker/) for sufficient contrast to pass WCAG AA.
- If it fails, a new shade of blue is implemented that does pass WCAG AA - the advice is to look at a lighter shade of dark blue.

**Comment by anaddledrabbit** [comment]
I put this on hold whilst I’m prioritising reviewing the column width stuff
